internal/mempool/mempool_test.go

```diff
@@ -2549,3 +2549,130 @@ func TestHandlesTSpends(t *testing.T) {
 		t.Fatalf("OnTSpendReceived callback was not called")
 	}
 }
+
+// TestHandlesTAdds verifies that the mempool correctly adds and removes valid
+// tadds.
+func TestHandlesTAdds(t *testing.T) {
+	t.Parallel()
+
+	// TODO: switch to mainnet params once they have treasury params set so
```

Comment:
Reminder here so that we can do this before merging once the mainnet parameters are set.

Same caveat applies here regarding the Pi key as the previous comment on a previous commit.

Reply:
The parameters (with the exception of the Pi key which can't be used here anyway as previously noted) have been added in #2170, so this can be updated accordingly.